src/Type/Definition/Type.php

```diff
@@ -133,22 +114,20 @@ public static function builtInTypes(): array
      */
     public static function getStandardTypes(): array
     {
-        return [
-            self::INT => static::int(),
-            self::FLOAT => static::float(),
-            self::STRING => static::string(),
-            self::BOOLEAN => static::boolean(),
-            self::ID => static::id(),
-        ];
+        return DefaultStandardTypeRegistry::instance()->standardTypes();
     }
 
     /**
+     * @deprecated use a custom instance of `DefaultStandardTypeRegistry` on the `Schema` instead
```

Comment:
Should we deprecate methods such as `getStandardTypes()` in this class too?

Reply:
Done, also deprecated the other static helpers like `Type::string`. Do you agree with that?